Fix Yelu Dynasty and lands. Fix Naiman Dynasty (not land yet). Removed duplicated Fergana and Urgench.

OnionMoeCat opened the pull request:
Still, the mongols seems to be in a mess because of duplicated characters/dynasties and messy government type. What should the mongol's land be?

Please note in the front page of the mod that I borrowed some history facts from:
[MOD]Horse Lords Remake+New Mongolian/Khitan/Han Portraits & Units
https://forum.paradoxplaza.com/forum/index.php?threads/mod-horse-lords-remake-new-mongolian-khitan-han-portraits-units.871973/#post-19669175

OnionMoeCat committed:
remove lands from kara-khitan and fix some title bugs for Yelu.

reason: Kara-Khitan is country name not dynasty name. Also, Kara-Khitan is built by Yelu family who escaped from China. I assign Kara-Khitan's land to Yelu.

OnionMoeCat committed:
fix duplicated urgench

OnionMoeCat committed:
Fix khitan history files. Also, please note I borrowed some history settings from Horse Lord Remake Mod.

OnionMoeCat committed:
Copy ck2 official file to folder.

OnionMoeCat committed:
Fix Naiman Dynasty characters.

Use vanilla Naiman Dynasty COA
Remove Liao/Kara-Khitai.

JFDCamara commented:
did you ask permission from that mod to be used? we can only use stuff from them if they gave authorization :)

JFDCamara commented:
BTW I added you as collaborator so you can edit the files directly

OnionMoeCat commented:
Thank you very much! But I prefer fork and pull request because my change
will be reviewed before adopted. Many errors could be picked out and fixed
in this way. I am a programmer and my team use this way to manage git.
Thank You!
OnionMoeCat

On Fri, Aug 21, 2015 at 2:10 PM, JFDCamara <notifications@github.com> wrote:

> BTW I added you as collaborator so you can edit the files directly
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JFDCamara/Umbra-Spherae-versions/pull/25#issuecomment-133547534>
> .
>
